invokeai/frontend/web/src/features/nodes/types/constants.ts

```diff
@@ -18,6 +18,8 @@ export const FIELD_TYPE_MAP: Record<string, FieldType> = {
   ColorField: 'color',
   ControlField: 'control',
   control: 'control',
+  cfg_scale: 'float',
```

Comment:
@psychedelicious Is this the way to handle this?

Reply:
Resolving to enable merge -- can look into alternatives later.